mycroft/skills/settings.py

```diff
@@ -228,6 +290,18 @@ def _is_new_hash(self, hashed_meta):
             return False if current_hash == str(hashed_meta) else True
         return True
 
+    def update_remote(self):
+        """ update settings state from server """
+        skills_settings = None
+        if self.get('is_synced'):
+            skills_settings = self._request_other_settings(hashed_meta)
+        if not skills_settings:
+            skills_settings = self._request_my_settings(hashed_meta)
+
+        if skills_settings is not None:
```

Comment:
could be simplified to
`else:`

Reply:
The control flow here is actually needed. the first if statement checks for skill settings that belong to other devices it can sync to. It can return None. the second if statements check to see if the skill settings exist for itself. It can also return None. if both returns None then don't save the settings

Reply:
Right right, so it could be `if skill_settings`